OG_Format.adoc

```diff
@@ -764,6 +764,15 @@ Byte <PARAM>_QC(N_MEASUREMENT); <PARAM>_QC:long_name = "quality flag";
 Note: It is anticipated to upgrade the ancillary variable related to QC by refining the ancillary variable name like < PARAM >_qc_generic, < PARAM >_qc_spike_test, <PARAM>_qc_land_test, etc. Current _QC attributes based on CF guidance (https://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#flags)
 and use the GOOS networks 0-4 flagging convention.
 
+////
+* [[depth-average-currents]]
+= Depth average currents
+////
+== Depth average currents
+
+Depth average currents should be included as a geophysical value in the N_MEASUREMENTS dimension along with an associated QC channel at the time when the calculation was completed by the glider. The methodology for derivation of the depth average current can be included as a methodology attribute to the variable. Results of the OceanGliders working group on depth average currents are pending at the time of writing (25th April 2023). Once, the working group provides result the approach for inclusion of depth average currents should be reviewed.
```

Comment:
Using `N_MEASUREMENTS` would impose a lot of redundancy, right?

Reply:
@castelao Not sure what you mean by this. Do you mean as opposed to defining them as scalars? Can you explain? We discussed the pros and cons at length on the call today and came to agreement that defining these parameters using the unlimited record dimension provides the most flexibility for the data providers.  Yes, there will be lots of missing/_FillValues but compression will make the file size increase negligible.  There are some cons (ERDDAP aggregation, distribution, etc.) depending on whether the data providers are submitting one file/underwater segment as opposed to one file that contains all of the underwater segments.

Reply:
@kerfoot , let's first check if we are understanding the same from the proposed text. My understanding is that as temperature is defined as `TEMP(N_MEASUREMENTS)`, so would be `depth_averaged_velocity(N_MEASUREMENS)`. Thus, even though such an average is only possible between two surfaces, the same value would be repeated as many times as temperature measurements during a dive, which could be on the order of thousands. Did I understand it correctly?

Reply:
No. The calculated depth-average current value would be reported at the appropriate time(N_MEASUREMENTS) row only. The time of the reported depth-averaged current would be left up to the data provider, according to their own determination. While the underlying calculation of the depth-averaged current is likely similar for all glider types, the timing of the reported current will be left up to the data provider.

Reply:
I'm still confused, but I think we are getting there. So the `depth_averaged_velocity` is not a scalar, but has the same dimension of time, which is `N_MEASUREMENTS`. Only one value of `depth_averaged_velocity` would be reported, which would be aligned with the respective time that the average is representative for. Is it the proposed idea?

Reply:
I think I understand the confusion.  We tried to explain this on the call this morning but unfortunately, the discussion was cut short.  It depends on whether files stored in the OG1.0 format are 1) a single file in which all underwater segments from the entire deployment are included or 2) multiple files, one from each underwater segment.  If it's #1, then use of a scalar (or another variable with an additional dimension) presents problems with reporting all depth-average currents. In this case, these values should be contained in TIME_UV(N_MEASUREMENTS), LAT_UV(N_MEASUREMENTS), etc., variables with _FillValues where not current is reported.  If it's #2 and there is only 1 depth-averaged current per file, imo it makes much more sense to me to store them in scalars, which would not require an additional dimension and would allow for erddap aggregation. This is the approach we take at the IOOS Glider DAC and it works very well.

Reply:
@castelao I'll offer my take as well. I think your last comment matches what @kerfoot proposed on the call. As I understood it, assuming the file contains multiple profiles, the suggestion was to include a sparse depth averaged current variable where there are mostly fill values but one depth averaged current value per profile. @kerfoot proposed file size would not be an issue if nc compression is used. An alternative is to follow the NGDAC approach where each .nc file is one indiv. underwater segment. In this case the depth averaged velocity can be stored as a scalar. I don't personally have experience aggregating scalars like this but @kerfoot reports this works well.

Reply:
HI all, just to remind everyone, the main intended use of OG1.0 is for the fundamental data object of "mission" a.k.a. "deployment." So I believe the proposal we have to consider is whether to include DAC as a variable with same dimensions as the fundamental geophysical dimension (N_MEASUREMENTS), with FillValue on most points. I am fine leaving this to the next version, and then we could add surface velocity as well at the same time. My reason for pushing this is because all the glider types have been computing this for years and the data are there for the taking already (with many caveats).

Reply:
If I'm the only one seeing an issue here, feel free to 'resolve conversation'.

Reply:
Hi all, I think we should leave this issue until the next version. Looking at the manual as a whole, a dedicated, but cryptic, section devoted to DAC seems out of place. After all, we have the generic <PARAM> and reference to vocabularies in the Geophysical Variables section and do not prescribe that any particular quantity should be included, only how it should be done. Since current is in the vocab, users can do it now as they wish, or wait until the next version for more guidance. this discussion above will be a good starting point, and is here for posterity.